lib/layout/box_layout.dart

```diff
@@ -202,7 +202,8 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
       List<Widget> updatedChildren = [];
       for (var i=0; i<children.length; i++) {
         updatedChildren.add(children[i]);
-        if (i != children.length-1) {
+        final child = children[i] as BoxLayout;
```

Comment:
Is it safe to downcasting the child to BoxLayout type right? @vusters I hope the gap property is only for BoxLayout and its subclasses like Row, Column and Flex widgets